### ISSUE TITLE
stand by, for libcdio 

### ISSUE THREAD

**Issue by kuboosoft** [UnitedRPMs/libcdio#1]
libcdio,was updated; isn't necessary have the 'rpm' in united rpm; only maintain the githug repo for testing/an-updated purposes.

**Comment by sergiomb2** [comment]
yeah is updated in every branches https://apps.fedoraproject.org/packages/libcdio
